Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'bootsnap', require: false
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223793885-eb23cf0f-239b-490f-b34f-17dcd712d448.png)

**[Optional]**

- [ ] It would be nice if your users and post page show the right number of posts from a user because i have created two posts from both users but i can only see zero posts :fearful: Kindly fix it :blush: